src/Components/SmartComponents/SelectAllCheckbox/SelectAllCheckbox.js

```diff
@@ -77,16 +77,17 @@ class SelectAllCheckbox extends Component {
 
     render() {
         const { selectedItems, cves } = this.props;
-        const { meta } = cves;
+        const { meta, data } = cves;
         const { isOpen } = this.state;
         const checkboxlabel = Boolean(selectedItems) && `${selectedItems} selected`;
+        const selectPageNumber = data.length / 2;
 
         const dropdownItems = [
             <DropdownItem key="none" onClick={this.handleUnselectAll}>
                 Select none (0 items)
             </DropdownItem>,
             <DropdownItem key="page" onClick={this.handleSelectPage}>
-                Select page ({meta.page_size} items)
+                Select page ({selectPageNumber} items)
```

Comment:
it's minor but in the case of displaying only one item, the verbal `items` will be displayed in plural

Reply:
same here

Reply:
Fixed